backend/src/main/java/com/sep6/backend/security/config/JwtAuthenticationFilter.java

```diff
@@ -35,7 +35,7 @@ protected void doFilterInternal(
             @NonNull FilterChain filterChain
     ) throws ServletException, IOException {
         log.info("Filtering request: {}", request.getServletPath());
-        if (request.getServletPath().contains("/auth")) {
+        if ((!request.getServletPath().matches("/accounts/.*")) || (!request.getServletPath().matches("/movies/[^/]+/reviews"))) {
```

Comment:
What is the if statement for? :smile:

Reply:
It is so we do not validate the token on endpoints that do not need authentication